amplitude.js

```diff
@@ -178,6 +178,9 @@ Amplitude.prototype.init = function(apiKey, opt_userId, opt_config) {
       if (opt_config.domain !== undefined) {
         this.options.domain = opt_config.domain;
       }
+      if (opt_config.includeUtm !== undefined) {
```

Comment:
No. I was just following convention in the file. Makes it less likely to break if you change the default to true in the future.